docs/configure_rum_android_sdk.md

```diff
@@ -0,0 +1,179 @@
+| `useViewTrackingStrategy(strategy)` | Defines the strategy used to track Views. Depending on your application's architecture, you can choose one of several implementations of [`ViewTrackingStrategy`][3] or implement your own. |
+| `addPlugin(DatadogPlugin, Feature)`   | Adds a plugin implementation for a specific feature (CRASH, LOG, TRACE, RUM). The plugin will be registered once the feature is initialized and unregistered when the feature is stopped. |
+ 
+### Automatically Track Views
+
+To automatically track your views (activities, fragments etc), provide a tracking strategy at initialization. Depending on your application's architecture, you can choose one of the strategies:
+
+| Strategy | Description                                                                                                                                                                                                                                                   |
+|------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `ActivityViewTrackingStrategy`   | Every activity in your application is considered a distinct view. |
+| `FragmentViewTrackingStrategy`     | Every fragment in your application is considered a distinct view.    |
+| `MixedViewTrackingStrategy` | Every activity or fragment in your application is considered a distinct view.  |
+| `NavigationViewTrackingStrategy`| Recommended for Android Jetpack Navigation library users. Each Navigation destination is considered a distinct view.  |
+
+For instance to set each Fragment as a distinct view, use the following configuration in your [setup][1]:
+   
+```kotlin
+           
+          val configuration = Configuration.Builder()
+                           .useViewTrackingStrategy(FragmentViewTrackingStrategy)
+                           .build()
+          
+       }
+   }
+```
+   
+**Tip**: For `ActivityViewTrackingStrategy`, `FragmentViewTrackingStrategy`, or `MixedViewTrackingStrategy` you can filter which `Fragment` or `Activity` is tracked as a RUM View by providing a `ComponentPredicate` implementation in the constructor.
+   
+**Note**: By default, the library won't track any view. If you decide not to provide a view tracking strategy you will have to manually send the views by calling the `startView` and `stopView` methods yourself.
+
+
+### Automatically Track Network requests
+
+To get timing information in Resources (3rd party providers, network requests) such as time to first byte, DNS resolution, etc., customize the okHttpClient to add the [Event listener][8] factory:
+
+    ```kotlin
+    val okHttpClient =  OkHttpClient.Builder()
+        .addInterceptor(DatadogInterceptor())
+        .eventListenerFactory(DatadogEventListener.Factory())
+        .build()
+    ```
+
```

Comment:
We should add a section to describe how we can track long tasks: 
```suggestion
### Automatically Track Long Tasks

To track long running operations performed on the main thread, which can impact the visual performance and reactivity of your app, you can do so by defining the duration threshold above which a task will be considerd too long: 
    val config = Configuration.Builder(rumEnabled=true)
                        .trackLongTasks(<durationThreshold>)
                        .build()
```

Reply:
Is there a default duration threshold? Let's also include this in the `getting started` section as a default piece.